packages/libssp.rb

```diff
@@ -4,7 +4,17 @@
 class Libssp < Package
   description 'Libssp is a part of the GCC toolkit.'
   homepage 'https://gcc.gnu.org/'
-  version "14.2.0-glibc#{LIBC_VERSION}" # Do not use @_ver here, it will break the installer.
+  @gcc_libc_version = if %w[2.23 2.27 2.32 2.33 2.35 2.37].any? { |i| LIBC_VERSION.include? i }
+                        LIBC_VERSION
+                      else
+                        case ARCH
+                        when 'i686'
+                          '2.23'
+                        when 'x86_64', 'aarch64', 'armv7l'
```

Comment:
This line could also be replaced with `else`.

Reply:
Or you could just use tertiary logic?
```
ARCH.eql?('i686') ? '2.23' : '2.27'
```

Reply:
Ternary is definitely the way to go here.

Reply:
Yes, ternary is a better word than tertiary.